doc/usage.md

```diff
@@ -750,6 +750,12 @@ Immediately drop all client and server connections on given database.
 New client connections to a killed database will wait until **RESUME**
 is called.
 
+#### KILL_CLIENT [ptr]
+
+Immediately kill specificed client connection along with any server 
+connections for the given client. The client to kill, is identified
+by the ptr value that can be found using the SHOW CLIENTS command.
```

Comment:
I think this would benefit from a tiny example, to make it very clear what the pointer value is expected to look like.

Reply:
Agreed, non C devs who are pgbouncer end users might not understand or care what a pointer is.

Reply:
Addressed in last commit. Added another line with an example in a `` block. I think memory addresses are different lengths in windows vs linux but I think it should be apparent enough what to look for with this line.